### PR TITLE
feat: add home page and styling

### PR DESCRIPTION
**Pull request by ferbun-wirya**
* add home page 
* add basic css style

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="ku">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="her yêke û şitllêk biçênîn, Daristanêk pêk dênîn">
```
Comment (ferbun-gelawej) [inline index.html]:
```suggestion
    <meta name="description" content="Her yêke û şitllêk deçênîn, Daristanêk pêk dênîn">
```
Tikeye diruşimekey çak, be hele nûsirawe

File: index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    <title>Daristan</title>
+    <link rel="stylesheet" href="css/master.css">
+</head>
+<body>
+
```
Comment (ferbun-gelawej) [inline index.html:11]: **Her wek pêşniyarêk:** wa baştir bû şitêkiman danaba

**Review by ferbun-gelawej — CHANGES_REQUESTED** [review]
We have to change the slogan in the description.

File: index.html (hunk 2 above)
Comment (ferbun-wirya) [reply index.html:11]: @ferbun-gelawej  bellam bedaxewe nusirawem we gîr nekewto

**Review by ferbun-gelawej — APPROVED** [review]
Serkewtû bî cewan

**Pull request merged**